tests/QS/regtest-p-efield/TEST_FILES

```diff
@@ -8,6 +8,7 @@ H2O-field-gopt.inp                                     1      3e-11
 H2O-field-gopt-lsd.inp                                 1      4e-12             -16.84348812829990
 H2O-field.inp                                          1      4e-13             -15.90320150373305
 H2O-field-lsd.inp                                      1      4e-14             -15.92914916395207
+H20-field-list.inp                                     2      1e-14            -0.171095788220e+02
```

Comment:
This is the number zero instead of a capital letter O.

Reply:
oops, thank you